src/v2/view-builder/views/RedirectIdPView.js

```diff
@@ -1,6 +1,24 @@
 import IdentifierView from './IdentifierView';
+import IdentifierFooter from '../components/IdentifierFooter';
+import { getBackToSignInLink } from '../utils/LinksUtil';
+
 
 export default IdentifierView.extend({
+
+  initialize() {
```

Comment:
what is the remediation ? does the api response doesn't contain `cancel` object

Reply:
In my testing, it didn't render in either case (the remediation with or without the cancel object). It's because our logic flows through IdentifierFooter.js and this does not take that into account when generating the links: https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/components/IdentifierFooter.js#L34

Reply:
check if BaseFooter intentionally not adding it ?  or at `AppState.js` `shouldShowSignOutLinkInCurrentForm` returning false

Reply:
@pradeepdewda-okta Yeah it looks like `shouldShowSignOutLinkInCurrentForm` in `AppState` is returning false ultimately because: https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/RemediationConstants.js#L74. So we can simply just remove this from the list and it would do the job. Although I'm not sure why this was added here to begin with. The history doesn't reveal much.